package.json

```diff
@@ -70,6 +70,7 @@
     "webpack": "^4.28.3"
   },
   "dependencies": {
-    "prop-types": "^15.6.2"
+    "prop-types": "^15.6.2",
+    "string.prototype.matchall": "^4.0.2"
```

Comment:
It looks like `matchall` is polyfilled with corejs 3.4 but we use corejs 2 in bigmaven. :( 

I am going to see if we can bump it -- it might be locked due to our NodeJS version.

Reply:
It looks like this is more work that a quick upgrade: https://github.com/mavenlink/mavenlink/pull/21288 The extra features from core-js 3+ is creating a different testing environment. I think because of that, we will want to fix the tests and then let it bake on team stagings for 2 weeks.